### DISCUSSION TITLE
Description

### DISCUSSION THREAD

**Discussion by fahad86** [TeamNewPipe/NewPipe#11011] [Questions] [answered]
Is there a way to view the description of the videos or are there any plans of adding this feature. I can see comments are supported but surprised to not be able to find the description of the videos![Screenshot_20240428_164246_YouTube.jpg](https://github.com/TeamNewPipe/NewPipe/assets/4401946/4f408df8-7762-4c92-9707-8d4734c5ea4b)

**Comment by opusforlife2** [comment, marked as answer]
Swiping left and right below the video doesn't work for you?

**Reply by fahad86** [reply]
It does thanks.. Missing out a lot of things in Folds 3 small cover screen.. I guess it's already intuitive enough..
